templates/cemanager.yaml

```diff
@@ -100,11 +100,6 @@ spec:
               value: {{ .Values.logging.json | quote }}
             - name: STACKDRIVER_LOG
               value: {{ .Values.logging.stackdriver | quote }}
-              # ENVBUILDER_IMAGE describes the image used to build
-              # user images and populate them with coder assets
-              # (such as code-server).
-            - name: ENVBUILDER_IMAGE
```

Comment:
I guess cemanager did some magic to use this image (e.g. creating a separate pod spec using envbuilder) rather than us installing it via Helm?

Reply:
Yup!